boards/arc/qemu/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2020 Synopsys, Inc. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_QEMU
```

Comment:
actually this is wrong, it should be BOARD_QEMU_ARCEM or something, lets not take this general name and make it ARC specific.  So start by calling the folder qemu_arcem and add everything to be for that board, then add the second configuration

Reply:
Renamed to `BOARD_QEMU_ARC` as it supports both ARC EM & ARC HS cores simultaneously.